src/main/java/duke/taskclasses/TaskList.java

```diff
@@ -76,13 +79,14 @@ public int getTasksCount() {
      * @param isDone The completion status of the task.
      * @param description Description of the task.
      */
-    public void addToDoToList(Boolean isDone, String description) {
+    public Task addToDoToList(Boolean isDone, String description) {
```

Comment:
Maybe the adding specific types of tasks could be handled by the subclasses of Task instead?